Library/Formula/fpc.rb

```diff
@@ -28,11 +28,15 @@ def install
 
     # Prevent non-executable audit warning
     rm_f Dir[bin/"*.rsj"]
+
+    # Generate a default fpc.cfg to set up unit search paths
+    system "#{bin}/fpcmkcfg", "-p", "-d", "basepath=#{lib}/fpc/#{version}", "-o", "#{etc}/fpc.cfg"
   end
 
   test do
     hello = <<-EOS.undent
       program Hello;
+      uses GL;
```

Comment:
ok i see the problem, #{etc} is the wrong directory for this file: it is installed in /usr/local/etc/ but the binary is just a symlink elsewhere, so even if the compiler is in /usr/local/bin/, it can't find the configuration file in the directory above.

It needs to be #{prefix}/etc, which normally expands to /usr/local/Cellar/fpc/3.0.0/etc, I'll update the PR shortly